src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/RuntimeEventSource.cs

```diff
@@ -23,7 +23,7 @@ public static class Keywords
             public const EventKeywords ProcessorCount = (EventKeywords)0x2;
         }
 
-        internal static RuntimeEventSource? Log => new RuntimeEventSource();
```

Comment:
Also add a comment to prevent the regression happening again?

Reply:
What should the comment say? 

We have hundreds of static auto-implemented properties with initializers in the repo. I do not think any of them has a comment that says that it really needs to be an auto-implemented property.

Reply:
done